ui/src/main/webapp/app.js

```diff
@@ -16,6 +18,62 @@ const theme = createMuiTheme({
   },
 })
 
+const Banner = props => {
+  const { children } = props
+
+  return (
+    <Query query={getUiConfig}>
+      {({ data, loading, error }) => {
+        if (loading) return <p>Loading...</p>
+        if (error) return <p>Error...</p>
+
+        return (
+          <div>
+            <AppBar
+              position='static'
+              style={{
+                fontFamily: 'Sans-serif',
```

Comment:
I don't think we should override the font so that it is consistent with the font of the rest of the app.

Reply:
I changed the font family to try and make it look a bit more like the banners in the admin console. Here's a picture with the header in the original font
![replicationUIserif](https://user-images.githubusercontent.com/22778496/55041094-b2115c80-4fe8-11e9-9f0b-e85a892de698.PNG)
And here's a picture of the admin console.
![adminUI](https://user-images.githubusercontent.com/22778496/55041106-c48b9600-4fe8-11e9-90fc-96afb69a17ed.PNG)
I thought it looked a bit odd with the default font but I'll change it back if you prefer.

Reply:
I think we should keep the font consistent on the replication page, but doesn't really matter either way.